packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/main_executable/main.swift

```diff
@@ -0,0 +1,13 @@
+@_implementationOnly import SomeLibrary
```

Comment:
What does this do?
And perhaps add a comment explaining it?

Reply:
This is the new feature being tested. I don't think it makes sense to explain it here; eventually it'll be an actual Swift language feature.